src/hatch_openzim/files_install.py

```diff
@@ -93,9 +98,28 @@ def _process_one_action(
     else:
         raise Exception(f"Unsupported action '{action}'")
 
+    execute_after = action_data.get("execute_after", None)
+    if execute_after:
+        _process_execute_after(base_target_dir=base_target_dir, actions=execute_after)
+
     logger.info("    Done")
 
 
+def _process_execute_after(base_target_dir: Path, actions: List[str]):
+    """execute actions after file(s) installation"""
+
+    for action in actions:
+        logger.info(f"    Executing '{action}'")
```

Comment:
I think I already requested not to use check_output. Use `run()`.
I also believe I requested to use `text=True` as to remove the need for decode afterwards

Reply:
Sorry about that, I'm really confused about this `subprocess` module, I don't achieve to remember where we have fixed this to the proper way of doing things. At least now with this hatch plugin it will maybe be easier to remember.